semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -459,6 +459,10 @@ def resolve_ref(self, semconv):
                 attr.attr_type = ref_attr.attr_type
                 if not attr.brief:
                     attr.brief = ref_attr.brief
+                if not attr.requirement_level:
+                    attr.requirement_level = ref_attr.requirement_level
+                if not attr.requirement_level_msg:
```

Comment:
I think the message should only be inherited if the requirement_level is not overriden.